### PR TITLE
Some updates to scheduler v3

### PR DESCRIPTION
**Pull request by cqg2**

### CODE REVIEW

File: src/studying.md [hunk 1]
```diff
@@ -219,8 +219,6 @@ mainly left around for compatibility with older Anki versions.
 
 - **Delete Note**: Deletes the note and all of its cards.
 
-- **Options**: Edit the options for the current deck.
```
Comment (dae) [inline src/studying.md:222]: Why remove this?

File: src/filtered-decks.md [hunk 2]
```diff
@@ -249,12 +256,12 @@ By default, Anki will return cards to their home decks with altered scheduling,
 based on your performance in the filtered deck. There is a **reschedule
 cards based on my answers** option available which alters this behaviour.
 
-In the [new scheduler](https://faqs.ankiweb.net/the-anki-2.1-scheduler.html),
+From [v2 scheduler](https://faqs.ankiweb.net/the-anki-2.1-scheduler.html),
```
Comment (dae) [inline src/filtered-decks.md:259]:
```suggestion
From the [v2 scheduler](https://faqs.ankiweb.net/the-anki-2.1-scheduler.html),
```

File: src/filtered-decks.md [hunk 3]
```diff
@@ -249,12 +256,12 @@ By default, Anki will return cards to their home decks with altered scheduling,
...
 when rescheduling is disabled, cards are shown in a simple "preview mode", and
 when they are returned to the original deck, they are returned exactly as they
 started.
 
-In the old scheduler, cards were presented more similarly to how they normally
+In v1 scheduler, cards were presented more similarly to how they normally
```
Comment (dae) [inline src/filtered-decks.md:264]:
```suggestion
In the v1 scheduler, cards were presented more similarly to how they normally
```

File: src/filtered-decks.md [hunk 4]
```diff
@@ -184,12 +185,18 @@ Cards return to their home deck when (re)learning is complete. Thus if
...
 
-In the old scheduler, the **custom steps** option allows you to override the
+In the v1 scheduler, the **custom steps** option allows you to override the
 home deck’s steps and provide your own steps instead. The provided steps apply to both
 cards being learnt, lapsed reviews, and reviews ahead of time. Please note, however, that
-in the [new scheduler](https://faqs.ankiweb.net/the-anki-2.1-scheduler.html)
+from the [v2 scheduler](https://faqs.ankiweb.net/the-anki-2.1-scheduler.html)
 filtered decks no longer support custom steps.
 
+### [Scheduler v3](https://faqs.ankiweb.net/the-2021-scheduler.html)
```
Comment (dae) [inline src/filtered-decks.md:194]: These headings make the table of contents a bit messy - I would probably opt to make them `**bold**` instead.

Screen Shot 2021-08-03 at 11 28 20 am

**Review by dae — COMMENTED** [review]
Thanks cqg!

File: src/studying.md (hunk 1 above)
Comment (cqg2) [reply src/studying.md:222]: Because it appears twice in the manual.

Comment (dae) [reply src/studying.md:222]: Ah sorry, I did not see that in the diff context.

File: src/filtered-decks.md (hunk 4 above)
Comment (cqg2) [reply src/filtered-decks.md]: Done

File: src/studying.md (hunk 1 above)
Comment (cqg2) [reply src/studying.md:222]: No problem :)

File: src/filtered-decks.md [hunk 5]
```diff
@@ -184,12 +186,20 @@ Cards return to their home deck when (re)learning is complete. Thus if
...
 home deck’s steps and provide your own steps instead. The provided steps apply to both
 cards being learnt, lapsed reviews, and reviews ahead of time. Please note, however, that
-in the [new scheduler](https://faqs.ankiweb.net/the-anki-2.1-scheduler.html)
+from the [v2 scheduler](https://faqs.ankiweb.net/the-anki-2.1-scheduler.html)
 filtered decks no longer support custom steps.
 
+**Scheduler v3**
+
+Filtered decks with rescheduling disabled show 4 buttons in the 
+[Scheduler v3](https://faqs.ankiweb.net/the-2021-scheduler.html) - the provided
```
Comment (dae) [inline src/filtered-decks.md:198]:
```suggestion
[v3 scheduler](https://faqs.ankiweb.net/the-2021-scheduler.html) - the provided
```

**Review by dae — COMMENTED** [review]
Thanks, just one minor thing I missed. I'll see if I can apply the change myself.

**Comment by dae** [comment]
All set - thanks!

**Pull request merged**